corporate/lib/stripe.py

```diff
@@ -3119,6 +3120,10 @@ def write_to_audit_log(
             "remote_realm": self.remote_realm,
             "event_type": audit_log_event,
             "event_time": event_time,
+            # At most one of these should be set, but we may
+            # not want an assert for that yet:
+            "acting_support_user": self.support_staff,
+            "acting_remote_user": self.remote_billing_user,
```

Comment:
It may have been nicer to **explicitly** pass acting users as args to `self.write_to_audit_log`, but I don't think that can be done simply enough - from my reading of it, we have a bunch of places in **general** codepath, where it could be either a regular Cloud billing session or one of these new Remote billing session and it gets called in this style:
```
            self.write_to_audit_log(
                event_type=AuditLogEventType.CUSTOMER_SWITCHED_FROM_MONTHLY_TO_ANNUAL_PLAN,
                event_time=timezone_now(),
                extra_data={
                    "monthly_plan_id": plan.id,
                    "annual_plan_id": new_plan.id,
                },
            )
```

So we can't pass the remote acting user values explicitly without making ugly `if`s all over the place to distinguish remote from remote session case.

Reply:
And these `self.remote_billing_user`, `self.support_staff` values should be correct anyway - unless we make bugs when calling the class constructors, which is hopefully not too likely.

Reply:
I like this pattern better anyway; I think the only case where passing explicitly would make sense is to override the default for the session and make something `None` (which is really just relevant for the end-of-cycle updates, where something that would usually happen in a background job might be processed under the current user's billing session as part of trying to show that user up-to-date data)